src/App.tsx

```diff
@@ -2,104 +2,22 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { PostList } from './components/PostList';
+
+const preparedPosts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => user.id === post.userId || null),
```

Comment:
`null` портрібен замість результата `find`, якщо нічого не знайдено
А не в середині колбека що перевіряє
```suggestion
  user: usersFromServer.find(user => user.id === post.userId) || null,
```